src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -78,10 +80,12 @@ export function GaugeChartProvider(Private) {
             self.addEvents(gauges);
           });
 
-          div.append('div')
-            .attr('class', 'chart-title')
-            .style('text-align', 'center')
-            .text(data.label || data.yAxisLabel);
+          if (self.gaugeConfig.type !== 'simple') {
```

Comment:
Why are you omitting the `chart-title` div when the type is not `simple`?

Reply:
to minimise the differences with the old pre 5.5 metric (and 6.1)

Reply:
with 6.1 we are loosing this whole gauge `simple` type (we are bringing the old metric vis back which is not SVG based)